libs/community/langchain_community/utilities/passio_nutrition_ai.py

```diff
@@ -0,0 +1,132 @@
+"""Util that invokes the Passio Nutrition AI API.
+"""
+from datetime import datetime, timedelta
+from typing import Union, Dict, List, final
+
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
can we remove this? dotenv isn't a langchain-community dependency

Reply:
Sure.

Reply:
Fixed it and push the changes.